Translation corrections (#77)
07th-mod/higurashi-console-arcs: Update/some_004.txt

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "とりあえず、…千紗登ちゃんと珠子ちゃんがあらかじめアドバイスしてくれたおかげで、少なくとも言葉に詰まることなく問答ができている。これも千紗登ちゃんたちのおかげだと、そっと内心で感謝した。",
				NULL, "<size=-2>For the time being... thanks to Chisato-chan and Tamako-chan's advice, I was at least able to answer the questions without getting stuck on the words. I owed that to the both of them, so I mentally thanked them.", Line_Normal);
	} else {
		OutputLine(NULL, "とりあえず、…千紗登ちゃんと珠子ちゃんがあらかじめアドバイスしてくれたおかげで、少なくとも言葉に詰まることなく問答ができている。これも千紗登ちゃんたちのおかげだと、そっと内心で感謝した。",

> 				NULL, "For the time being... thanks to Chisato-chan and Tamako-chan's advice, I was at least able to answer the questions without getting stuck on the words. I owed that to Chisato-chan, so I thanked her from the bottom of my heart.", Line_Normal);

Reviewer: ```suggestion
				NULL, "For the time being... thanks to Chisato-chan and Tamako-chan's advice, I was at least able to answer the questions without getting stuck on the words. I owed that to the both of them, so I mentally thanked them.", Line_Normal);
```

//rそして、尾形先生は。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、尾形先生は。",
			NULL, "And then...", Line_Normal);
	ClearMessage();

//rファイルの下にあった別のファイルを取り出して、私に広げてみせる。グラフ仕立てで色々な数字が書かれたそこには、…私の成績と、大学の合格ラインのリストがあって。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ファイルの下にあった別のファイルを取り出して、私に広げてみせる。グラフ仕立てで色々な数字が書かれたそこには、…私の成績と、大学の合格ラインのリストがあって。",

> 			NULL, "Ogata-sensei pulled another file out from underneath the other one and spread it out for me. There were various graphs drawn there... My grades were listed there along with a list of university acceptance scores.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Ogata-sensei pulled another file out from beneath the first one and spread it out for me. There were various graphs drawn there... My grades were listed there along with a list of university acceptance scores.", Line_Normal);
```
